pkg/pkg.mk

```diff
@@ -13,6 +13,7 @@ GIT_APPLY_PATCHES:=if test -d "$(PKG_DIR)"/patches; then \
 	fi
 
 $(PKG_BUILDDIR)/.git-downloaded:
+	rm -Rf $(PKG_BUILDDIR)
```

Comment:
>at least for the scope of what we want to achieve (don't fail because directory already exists) ;-)

I understand that we **want** to delete the whole directory tree. Just `mkdir -p` won't work then.